src/routes/safe/components/Transactions/TxsTable/ExpandedTx/OwnersColumn/ButtonRow.jsx

```diff
@@ -20,6 +20,9 @@ const styles = () => ({
   },
   button: {
     height: xl,
+    '&:hover': {
```

Comment:
I think this has to be overwritten for every danger button, so please update the material ui theme (src/theme/mui.js)

Reply:
Done